website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -0,0 +1,593 @@
+---
+page_title: Terraform Cloud Operator for Kubernetes API reference
+description: >-
+  API reference for the Terraform Cloud Operator for Kubernetes
+---
+
+# API Reference
```

Comment:
This is a pretty long page that feels hard to navigate right now.

 I think it'd be more helpful to have the attributes like `AgentDeployment` and `AgentDeploymentAutoscaling` be h2s on this page so you can see them in the right-hand TOC on the site? Since this is a very referential page, I'm assuming folks will come here to look up a specific attribute and having them all in the TOC would make lookup way easier.

Reply:
Or we could add guidance to an intro paragraph of how best to navigate this page?

Reply:
Ooooh or if we could do a Resource types list like in the Vault API page that'd also help:
https://github.com/hashicorp/vault/blob/dd15e5296f1ad6fb64a1deb92dc307bb9c1e275d/website/content/docs/platform/k8s/vso/api-reference.mdx#L22